### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by favour-olumese**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -9,15 +9,15 @@ HashR allows you to build your own hash sets based on your data sources. It's a
 HashR consists of the following components:
 
 1. Importers, which are responsible for copying the source to local storage and doing any required preprocessing.
-1. Core, which takes care of extracting the content from the source using image_export.py (Plaso), caching and repository level deduplication and preparing the extracted files for the exporters.
-1. Exporters, which are responsible for exporting files, metadata and hashes to given data sinks. Currently the main exporter is the Postgres exporter.
+2. Core, which takes care of extracting the content from the source using image_export.py (Plaso), caching and repository level deduplication and preparing the extracted files for the exporters.
```
Comment (meeehow) [inline readme.md:12]: Please revert list numbering changes, this syntax is supported by markdown (see https://www.markdownguide.org/basic-syntax/#ordered-lists) and I think it's easier to insert an element into the list this way.

File: readme.md [hunk 2]
```diff
@@ -274,13 +274,13 @@ This importer will extract files from all images it can find in the `install.wim
...
-1. Configure WSUS to automatically approve and download updates to local storage 
-1. Set up a Windows task to automatically sync content of the local storage to the GCS bucket: `gsutil -m rsync -r D:/WSUS/WsusContent gs://hashr-wsus/` (remember to adjust the paths)
+2. Configure it with WSUS role, select Windows Update packages that you'd like to process
+3. Configure WSUS to automatically approve and download updates to local storage 
+4. Set up a Windows task to automatically sync content of the local storage to the GCS bucket: `gsutil -m rsync -r D:/WSUS/WsusContent gs://hashr-wsus/` (remember to adjust the paths)
 
 ### Setting up Postgres exporter 
 
-Postgres exporter allows to send hashes, file metadata and the actual content of the file to a PostgreSQL instance. For best performance it's advised to set it up on a separate and dedicated machine. 
+Postgres exporter allows sending of hashes, file metadata, and the actual content of the file to a PostgreSQL instance. For best performance it's advised to set it up on a separate and dedicated machine.
```
Comment (meeehow) [inline readme.md:283]: I don't think we put comma before "and" in English in this context, ref https://www.grammarly.com/blog/comma-before-and/

**Review by meeehow — COMMENTED** [review]
Thanks for fixing spelling and grammar errors! I just have two comments here.

**Review by MujeebullahKalwar — APPROVED** [review]

**Review by meeehow — CHANGES_REQUESTED** [review]
Please take a look at two of my comments.

File: readme.md (hunk 1 above)
Comment (favour-olumese) [reply readme.md]: Good day. I have reverted the list numberings.

File: readme.md (hunk 2 above)
Comment (favour-olumese) [reply readme.md]: All right. I have removed the comma. Thank you.

**Review by meeehow — APPROVED** [review]

**Pull request merged**